SU-HSDO/suhumsci
# SHS-5503: Color Band Link hover state in Colorful theme

# READY FOR REVIEW

## Summary
Fix color and add underline to hover in colorful theme

## Need Review By (Date)
03/22

## Urgency
medium

## Steps to Test
1. Setup `hs_colorful` site. Log in and visit https://hs-colorful.suhumsci.loc/components/color-bands
2. Edit the page and add a link to all of the color bands in the page
3. For all color bands confirm the following:
  a. Links (both default and hover) are white, except for the `light` variable, which is black
  b. All links are underlined on hover
4. Visit the [theme settings page](https://hs-colorful.suhumsci.loc/admin/appearance/settings/humsci_colorful) and change the Color Pairing. Repeat the previous tests and confirm that that the link colors are the same
5. Repeat for all existing Color Pairings.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
